Rmd/DESeq2_report_new.Rmd

```diff
@@ -359,10 +349,10 @@ plot_DEG_heatmap <- function(rld_data){
   if (nrow(as.matrix(mat)) > 1) {
     mat <- mat - rowMeans(mat)
     return(pheatmap(mat,
-             annotation_col =  as.data.frame(colData(rld_DEGs)[, intgroup, drop = F]),
+             annotation_col =  as.data.frame(colData(rld_DEGs)[, params$intgroup_to_plot, drop = F]),
              show_rownames = (nrow(rld_data) <= 50),
              border_color = NA,
-             scale = "row"))
+             scale = "none"))
```

Comment:
Was there a reason for this change? Does it look better this way?

Reply:
I think the data is already scaled appropriately for this to look good, and this was causing it to fail if the data only had one sample (I think? Or one gene? one of something)

Reply:
Ok my comment wasn't quite correct. I don't remember exactly why I made this change so I've reverted it.